build_projects/update-dependencies/update-dependencies.ps1

```diff
@@ -34,12 +42,17 @@ if($LASTEXITCODE -ne 0) { throw "Failed to install stage0" }
 # Put the stage0 on the path
 $env:PATH = "$env:DOTNET_INSTALL_DIR;$env:PATH"
 
+# Generate some props files that are imported by update-dependencies
```

Comment:
For the rest of the internet, it continues to be needed because update-dependencies.csproj has a P2P reference to Microsoft.DotNet.Cli.Utils/Microsoft.DotNet.Cli.Utils.csproj, so I left this in.